modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptNodeClientCodegen.java

```diff
@@ -162,7 +163,7 @@ public String toModelImport(String name) {
             return importMapping.get(name);
         }
 
-        return modelPackage() + "/" + camelize(toModelName(name), true);
+        return DEFAULT_MODEL_IMPORT_DIRECTORY_PREFIX + modelPackage() + "/" + camelize(toModelName(name), true);
```

Comment:
@mahirk not a blocker on this pr, but wanted to comment since you said you were planning to look at ts implementation in other generators...

since modelPackage is passed by the user, this could result in `models//` when passed with trailing slash. It might be best to use StringUtils, something like:

```
return DEFAULT_MODEL_IMPORT_DIRECTORY_PREFIX + 
    StringUtils.appendIfMissing(modelPackage(), "/") +
    camelize(toModelName(name), true);
```